sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb

```diff
@@ -0,0 +1,1337 @@
+        "        ColumnTransformer(fields=[\"Quantity\"], parameters={\"strategy\": \"constant\", \"fill_value\": 0}),\n",
+        "        # Fill missing values in the INCOME column with median value.\n",
+        "        ColumnTransformer(fields=[\"INCOME\"], parameters={\"strategy\": \"most_frequent\"}),\n",
+        "        # Fill missing values in the Price column with forward fill (last value carried forward).\n",
+        "        ColumnTransformer(fields=[\"Price\"], parameters={\"strategy\": \"ffill\"}),\n",
+        "    ]\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Create the AutoML forecasting job."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "gather": {
+          "logged": 1634852262026
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "name": "forecasting-configuration",
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "forecasting_job = automl.forecasting(\n",
+        "    compute=cluster_name,\n",
+        "    experiment_name=exp_name,\n",
+        "    training_data=my_training_data_input,\n",
+        "    target_column_name=target_column_name,\n",
+        "    primary_metric=\"NormalizedRootMeanSquaredError\",\n",
+        "    n_cross_validations=\"auto\",\n",
+        "    enable_model_explainability=True,\n",
+        ")\n",
+        "\n",
+        "# Limits are all optional\n",
+        "forecasting_job.set_limits(\n",
+        "    timeout_minutes=60,\n",
+        "    trial_timeout_minutes=20,\n",
+        "    max_trials=max_trials,\n",
+        "    enable_early_termination=True,\n",
+        ")\n",
+        "\n",
+        "# Specify the above custom featurization\n",
+        "forecasting_job.set_featurization(\n",
+        "    mode=\"custom\",\n",
+        "    column_name_and_types=column_name_and_types,\n",
+        "    transformer_params=transformer_params\n",
+        ")\n",
+        "\n",
+        "# Specialized properties for Time Series Forecasting training\n",
+        "forecasting_job.set_forecast_settings(\n",
+        "    time_column_name=time_column_name,\n",
+        "    forecast_horizon=forecast_horizon,\n",
+        "    time_series_id_column_names=time_series_id_column_names,\n",
+        "    frequency=\"W-THU\" # Set the forecast frequency to be weekly (start on each Thursday)\n",
+        ")\n",
+        "\n",
+        "# Training properties are optional\n",
+        "forecasting_job.set_training(blocked_training_algorithms=[\"ExtremeRandomTrees\"])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 4.2 Run the Command\n",
```

Comment:
Please replace with `4.2 Train the AutoML model`.

Reply:
Done.